mod-audit-server/pom.xml

```diff
@@ -89,6 +89,11 @@
       <artifactId>json-path</artifactId>
       <version>${json.path.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>postgres-testing</artifactId>
+      <version>33.0.0</version>
```

Comment:
Please, move this version to properties.

Reply:
Done.